docs/Google-analytics.md

```diff
@@ -2,7 +2,9 @@
 
 ## Introduction
 
-[www.google.com/analytics](https://www.google.com/analytics) (GA for short) is a great (and free) way to see how many people are hitting your site. By default uportal-app-framework disables GA. However, if you follow the config setup below, you will get reporting.
+[www.google.com/analytics](https://www.google.com/analytics) (GA for short) is
```

Comment:
:art: if its going to be a link it may be worth considering
`[www.google.com/analytics](https://www.google.com/analytics)` :arrow_right: `<https://www.google.com/analytics>`